bedrock/firefox/templates/firefox/installer-help.html

```diff
@@ -20,56 +20,97 @@
   {% endwith %}
 {% endblock %}
 
+{% if installer_channel == 'beta' %}
+  {% set channel_logo = 'protocol/img/logos/firefox/browser/beta/logo-word-hor-sm.png' %}
+{% elif installer_channel == 'alpha' %}
+  {% set channel_logo = 'protocol/img/logos/firefox/browser/developer/logo-word-hor-sm.png' %}
+{% elif installer_channel == 'nightly' %}
+  {% set channel_logo = 'protocol/img/logos/firefox/browser/nightly/logo-word-hor-sm.png' %}
+{% else %}
+  {% set channel_logo = 'protocol/img/logos/firefox/browser/logo-word-hor-sm.png' %}
+{% endif %}
+
 {% block content %}
-<main class="mzp-l-content mzp-has-sidebar mzp-l-sidebar-right">
-  <h2><a href="{{ url('firefox.new') }}">{{ high_res_img('protocol/img/logos/firefox/browser/logo-word-hor-sm.png', {'alt': 'Firefox', 'width': '347', 'height': '64'}) }}</a></h2>
-  <div class="mzp-l-main">
-    <header>
-      <h1 class="installer-help-title">{{ _('Your download was interrupted') }}</h1>
-      {% if installer_channel %}
-        <h2 class="installer-help-subtitle">{{ _('Click the download button on the right to continue.') }}</h2>
-      {% else %}
-        <h2 class="installer-help-subtitle">{{ _('Please choose a download to continue.') }}</h2>
-      {% endif %}
-    </header>
-    <aside class="mzp-c-emphasis-box">
-      <h3 class="installer-help-emphasis-title">{{ _('Need help installing?') }}</h3>
-      <p class="installer-help-emphasis-tagline"><a href="https://support.mozilla.org/kb/install-firefox-windows">{{ _('Visit support.mozilla.org') }}</a></p>
-    </aside>
-  </div>
+<main class="mzp-l-content">
+  <section class="mzp-c-hero">
+    <div class="mzp-c-hero-body">
+      <h1 class="mzp-c-hero-title">{{ _('Hmm… your download was interrupted') }}</h1>
```

Comment:
Should this new design template be behind an L10 conditional in the view? Currently this PR is losing all existing translations, unless I'm misreading something?

Reply:
you are correct! I jumped the gun.